src/skin/skinloader.cpp

```diff
@@ -30,6 +30,15 @@ SkinLoader::~SkinLoader() {
 
 QList<QDir> SkinLoader::getSkinSearchPaths() const {
     QList<QDir> searchPaths;
+
+    // Add user skin path to search paths
+    QString userSkinsPath = m_pConfig->getSettingsPath();
```

Comment:
I would suggest to apply the same strategy for determining the sub-directory as for the following search paths, i.e. construct a QDir from the settings path and the try to cd into the "skins" sub-folder. Just for the sake consistency.

Why is it necessary to transform the path into an absolute path? I think this would only be needed for reproducible error reporting (see below).

Reply:
Sorry, I was being lazy and took the code from [the controller mapping implementation](https://github.com/mixxxdj/mixxx/blob/def83b701360edf1235a9e64491a459bd3683a28/src/controllers/defs_controllers.h#L27).

Reply:
Ok, I made the implementation more consistent.